pkgs/top-level/python-packages.nix

```diff
@@ -15315,16 +15439,15 @@ in {
   };
 
   pygal = buildPythonPackage rec {
-    version = "2.0.10";
-    name = "pygal-${version}";
+    pname = "pygal";
+    version = "2.3.1";
+    name = "${pname}-${version}";
 
     doCheck = !isPyPy;  # one check fails with pypy
```

Comment:
if its only one test, and the test is broken, then surely we can just disable that one test?